onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -66,13 +66,20 @@ struct BrokenTest {
   std::string test_name_;
   std::string reason_;
   std::set<std::string> broken_versions_ = {};  // apply to all versions if empty
+  std::set<std::string> broken_opset_versions_ = {};
```

Comment:
What if we remove the line 68? If we know the test case was wrong in an ONNX version, we might be able to modify our downloading script to just delete it after the downloading. My concern is if we have both opset version and onnx version here, it may confuse most developers. They don't know why both are needed

Reply:
since we would remove old /data/onnx, line 68 caould be removed now.
And the BrokenTest list might be affected by EPs, it couldn't be deleted after downloading

Reply:
Maybe we would leverage onnxruntime/test/testdata/...filters.jsonc in the next.